src/plugins/XKTLoaderPlugin/XKTLoaderPlugin.js

```diff
@@ -855,7 +855,7 @@ class XKTLoaderPlugin extends Plugin {
                 const lastEntity = (i === numEntities - 1);
                 const meshIds = [];
 
-                for (let j = entityMeshes [i], jlen = lastEntity ? entityMeshes.length : entityMeshes [i + 1]; j < jlen; j++) {
+                for (let j = entityMeshes [i], jlen = lastEntity ? entityMeshIds.length : entityMeshes [i + 1]; j < jlen; j++) {
```

Comment:
This leed needs NOT to be changed, as `entityMeshIds` only exists for XKT v2, and this line corresponds to XKT v1